tools/docs/github-query-items.ts

```diff
@@ -59,18 +70,21 @@ export async function getOpenGitHubItems(): Promise<RenovateOpenItems> {
     return result;
   }
 
-  const q = `repo:renovatebot/renovate type:issue is:open`;
-  const per_page = 100;
+  if (!process.env.GITHUB_TOKEN) {
+    logger.warn(
+      'No GITHUB_TOKEN found in env, cannot fetch Github issues. Skipping...',
+    );
+    return result;
+  }
+
+  if (process.env.CI) {
+    return result;
+  }
+
   try {
-    const query = getQueryString({ q, per_page });
-    const res = await githubApi.getJsonUnchecked<GithubApiQueryResponse>(
-      gitHubApiUrl + query,
-      {
-        paginationField: 'items',
-        paginate: true,
-      },
+    const rawItems: ItemsEntity[] = (await getIssuesByIssueType('Bug')).concat(
```

Comment:
```suggestion
    const rawItems = (await getIssuesByIssueType('Bug')).concat(
```
type should not needed here